include/kodlab_mjbots_sdk/soft_start.h

```diff
@@ -25,7 +25,7 @@ class SoftStart {
    * @param torques[in, out] the torques to be constrained
    * @param count the number of steps it has been
    */
-  void ConstrainTorques(std::vector<float> &torques, int count);
+  void ConstrainTorques(std::vector<float> &torques, float time_since_start_ms);
```

Comment:
fix comment

Reply:
Fixed in 21511f7a0d652fe35a9fc578d4b151ac0b83c9d4